### PR TITLE
[android] Update changelog for the v8.4.0-alpha.1 release

### PR DESCRIPTION
**Pull request by langsmith**
This PR updates the android changelog for releasing v8.4.0-alpha.1 (`ristretto`)

### CODE REVIEW

File: platform/android/CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,33 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
...
+
+### Features
+- Bindings for aggregated cluster properties [#15425](https://github.com/mapbox/mapbox-gl-native/pull/15425)
+- Add font array attribute configuration [#15488](https://github.com/mapbox/mapbox-gl-native/pull/15488) 
+- Expose camera padding option [#15444](https://github.com/mapbox/mapbox-gl-native/pull/15444) 
+- Add number format expression [#15424](https://github.com/mapbox/mapbox-gl-native/pull/15424) 
+
+### Bug fixes
+- Initialize CameraPositionActivity listeners after the style is loaded [#15484](https://github.com/mapbox/mapbox-gl-native/pull/15484)
+- Cancel CameraAnimatorActivity joint example when the map is destroyed [#15500](https://github.com/mapbox/mapbox-gl-native/pull/15500)
```
Comment (tobrun) [inline platform/android/CHANGELOG.md]: this is a testapp fix, no need to include this in sdk changelog

File: platform/android/CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,33 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
...
+[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.3.0...android-v8.4.0-alpha.1) since [Mapbox Maps SDK for Android v8.3.0](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.3.0):
+
+### Features
+- Bindings for aggregated cluster properties [#15425](https://github.com/mapbox/mapbox-gl-native/pull/15425)
+- Add font array attribute configuration [#15488](https://github.com/mapbox/mapbox-gl-native/pull/15488) 
+- Expose camera padding option [#15444](https://github.com/mapbox/mapbox-gl-native/pull/15444) 
+- Add number format expression [#15424](https://github.com/mapbox/mapbox-gl-native/pull/15424) 
+
+### Bug fixes
+- Initialize CameraPositionActivity listeners after the style is loaded [#15484](https://github.com/mapbox/mapbox-gl-native/pull/15484)
```
Comment (tobrun) [inline platform/android/CHANGELOG.md]: this is a testapp fix, no need to include this in sdk changelog

File: platform/android/CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,33 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
...
+- Cancel CameraAnimatorActivity joint example when the map is destroyed [#15500](https://github.com/mapbox/mapbox-gl-native/pull/15500)
+- Invalidate foreground icon sources even when location layer is hidden [#15507](https://github.com/mapbox/mapbox-gl-native/pull/15507)
+- Fixed unnecessary layers blink when the same components are re-added after the style reload [#15513](https://github.com/mapbox/mapbox-gl-native/pull/15513)
+- Fixed an issue in JNI marshaling code that converted a float to long that produced invalid expressions [#15557](https://github.com/mapbox/mapbox-gl-native/pull/15557)
+- Call callback onFinish() immediately when trying to move camera to current position [#15558](https://github.com/mapbox/mapbox-gl-native/pull/15558)
+
+### Build
+- Reorder android-check make command [#15493](https://github.com/mapbox/mapbox-gl-native/pull/15493)
```
Comment (tobrun) [inline platform/android/CHANGELOG.md]: this doesn't really impact normal usage, can be removed

File: platform/android/CHANGELOG.md [hunk 4]
```diff
@@ -15,6 +15,33 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
...
+- Ignore project wide gradle.properties file [#15495](https://github.com/mapbox/mapbox-gl-native/pull/15495)
```
Comment (tobrun) [inline platform/android/CHANGELOG.md]: this doesn't really impact normal usage, can be removed

File: platform/android/CHANGELOG.md [hunk 5]
```diff
@@ -15,6 +15,33 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
...
+
+### Performance improvements
+- JNI binding refactor [#15462](https://github.com/mapbox/mapbox-gl-native/pull/15462)
```
Comment (tobrun) [inline platform/android/CHANGELOG.md]: this refactor doesn't improve performance, it's a general refactor to make the code more modular

File: platform/android/CHANGELOG.md [hunk 6]
```diff
@@ -15,6 +15,33 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
...
+### Features
+- Bindings for aggregated cluster properties [#15425](https://github.com/mapbox/mapbox-gl-native/pull/15425)
+- Add font array attribute configuration [#15488](https://github.com/mapbox/mapbox-gl-native/pull/15488) 
+- Expose camera padding option [#15444](https://github.com/mapbox/mapbox-gl-native/pull/15444) 
+- Add number format expression [#15424](https://github.com/mapbox/mapbox-gl-native/pull/15424) 
+
+### Bug fixes
+- Initialize CameraPositionActivity listeners after the style is loaded [#15484](https://github.com/mapbox/mapbox-gl-native/pull/15484)
+- Cancel CameraAnimatorActivity joint example when the map is destroyed [#15500](https://github.com/mapbox/mapbox-gl-native/pull/15500)
+- Invalidate foreground icon sources even when location layer is hidden [#15507](https://github.com/mapbox/mapbox-gl-native/pull/15507)
```
Comment (LukasPaczos) [inline platform/android/CHANGELOG.md]: this shipped with 8.3.0

**Comment by LukasPaczos** [comment]
Also, It'd be great to use changelog entries provided [at the top of the changelog file ](https://github.com/mapbox/mapbox-gl-native/blob/20bef9f6f2ecb0be872149cbd5a32e6add5c58da/platform/android/CHANGELOG.md#master).

File: platform/android/CHANGELOG.md [hunk 7]
```diff
@@ -15,6 +15,33 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 ### Bug fixes
  - Fixed a rendering issue of `collisionBox` when `text-translate` or `icon-translate` is enabled. [#15467](https://github.com/mapbox/mapbox-gl-native/pull/15467)
 
+## 8.4.0-alpha.1 - September 4, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.3.0...android-v8.4.0-alpha.1) since [Mapbox Maps SDK for Android v8.3.0](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.3.0):
+
+### Features
+- Bindings for aggregated cluster properties [#15425](https://github.com/mapbox/mapbox-gl-native/pull/15425)
+- Add font array attribute configuration [#15488](https://github.com/mapbox/mapbox-gl-native/pull/15488)
```
[the remark this answers is not in the document]
Comment (langsmith) [reply platform/android/CHANGELOG.md]: Removed

File: platform/android/CHANGELOG.md (hunk 1 above)
Comment (langsmith) [reply platform/android/CHANGELOG.md]: Removed

File: platform/android/CHANGELOG.md (hunk 2 above)
Comment (langsmith) [reply platform/android/CHANGELOG.md]: Removed

File: platform/android/CHANGELOG.md (hunk 3 above)
Comment (langsmith) [reply platform/android/CHANGELOG.md]: Removed

File: platform/android/CHANGELOG.md (hunk 4 above)
Comment (langsmith) [reply platform/android/CHANGELOG.md]: Removed

File: platform/android/CHANGELOG.md (hunk 5 above)
Comment (langsmith) [reply platform/android/CHANGELOG.md]: Moved to `Other Changes` section

File: platform/android/CHANGELOG.md (hunk 6 above)
Comment (langsmith) [reply platform/android/CHANGELOG.md]: Removed

**Comment by langsmith** [comment]
> Also, It'd be great to use changelog entries provided at the top of the changelog file .

Ok @LukasPaczos , I've moved the entries in the `master` section to the respective sections in `8.4.0-alpha.1`. Fine to delete them from the `master` section?

File: platform/android/CHANGELOG.md (hunk 5 above)
Comment (LukasPaczos) [reply platform/android/CHANGELOG.md]: I don't think it's worth mentioning at all, it brings no relevant changes to the developers.

**Comment by LukasPaczos** [comment]
Yes @langsmith, totally fine to delete them.

File: platform/android/CHANGELOG.md [hunk 8]
```diff
@@ -25,7 +48,6 @@ This release changes how offline tile requests are billed — they are now bille
 
 ### Bug fixes
  - Fixed a rendering issue caused by all icons being treated as SDFs if an SDF and non-SDF icon were in the same layer. [#15456](https://github.com/mapbox/mapbox-gl-native/pull/15456)
- - Fixed an issue where changing location's render mode when the`LocationComponent` is disable wouldn't invalidate the foreground icon when it's back enabled. [#15507](https://github.com/mapbox/mapbox-gl-native/pull/15507)
```
Comment (LukasPaczos) [inline platform/android/CHANGELOG.md]: This was actually shipped with `8.3.0` and shouldn't be removed from here.

Comment (langsmith) [reply platform/android/CHANGELOG.md]: Whoops. Re-added.

**Comment by langsmith** [comment]
> Yes @langsmith, totally fine to delete them.

Ok, deleted.

**Pull request merged**